plugin/parser/differ/mysql/differ.go

```diff
@@ -0,0 +1,52 @@
+// Package mysql provides the MySQL differ plugin.
+package mysql
+
+import (
+	"bytes"
+
+	"github.com/pingcap/tidb/parser/ast"
+	"github.com/pingcap/tidb/parser/format"
+)
+
+// Differ is the differ plugin for MySQL.
+type Differ struct {
+	oldTables map[string]*ast.CreateTableStmt
```

Comment:
Is it necessary to have `oldTables` as a struct field vs. just a local variable?

In fact... it doesn't seem like we need a `Differ` struct at all? 🤔

Reply:
remove it.